packages/samples/mvp-dapp/kuai.config.js

```diff
@@ -1,4 +1,6 @@
+require('dotenv').config()
+
 module.exports = {
-  port: 3000,
-  rpcUrl: 'https://mainnet.ckbapp.dev/rpc',
+  port: process.env.PORT || 3000,
```

Comment:
`3000` is set as the default port, then `fallback to 3000` at https://github.com/ckb-js/kuai/blob/develop/packages/samples/mvp-dapp/src/main.ts#L14 is redundant